metadata-ingestion/src/datahub/ingestion/api/report.py

```diff
@@ -217,6 +219,7 @@ def __post_init__(self) -> None:
                 "entityType": lambda val: val.entity_type,
                 "subTypes": lambda val: val.subType,
                 "aspects": lambda val: json.dumps(val.aspects),
+                "is_stale": lambda val: val.is_stale,
```

Comment:
`is_stale` is our rationale (our excuse) to emit the soft deletion. Sort of business rule.

I would use here different terminology: `is_status_removed`, `is_soft_deleted`, or similar. Something more close to the entity model.